app/Http/Controllers/UpdateAirlineController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CreateAirlineRequest;
+use App\Models\Airline;
+use Illuminate\Http\JsonResponse;
+
+class UpdateAirlineController extends Controller
+{
+    public function __invoke(CreateAirlineRequest $request, Airline $airline): JsonResponse
```

Comment:
You're still using the `CreateAirlineRequest`. You should create and use a different Form Request that follows what's explained in [this](https://github.com/LukiMaurojorge/flight-agency/pull/2#discussion_r1526803980) comment.